samples/graph-teams-tag/nodejs/client/src/components/view-or-edit-tag.jsx

```diff
@@ -0,0 +1,282 @@
+// <copyright file="view-or-edittag.tsx" company="Microsoft Corporation">
```

Comment:
File name should be like view-edit-tag.jsx

Reply:
Done